pymare/estimators/combination.py

```diff
@@ -112,9 +112,9 @@ class StoufferCombinationTest(CombinationTest):
     # Maps Dataset attributes onto fit() args; see BaseEstimator for details.
     _dataset_attr_map = {"z": "y", "w": "v"}
 
-    def fit(self, z, w=None):
+    def _fit(self, z, w=None):
```

Comment:
Why make this private?

Reply:
Yeah, this one was tricky because `fit()` alone only works with datasets while `_fit()` is fitted to arrays. This was a solution that worked, but I'm sure there is a more elegant one that can make that distinction clearer.
Would you suggest a different approach? Thanks!

Reply:
To be honest, I don't remember if I had considered how fitting to arrays would work with the proposed changes... I wish I'd made a note about it in #103.

We don't want users to call a private method, so I think the options are to either make the array-fitting method a different public method (e.g., `fit_array`) or to support arrays in `fit`. WDYT?

Reply:
I think if `fit()` alone cannot distinguish between an array or dataset we should keep `fit()` only for arrays for the sake of scikit-learn styling, and `fit_dataset()` for datasets.
Does that mean we need to add `fit_dataset_transform()` in addition to `fit_transform()`?

Reply:
I think we want the Dataset to be the default (unless I'm misremembering), so I think `fit_array` and `fit` would be better than `fit` and `fit_dataset`. It's a difficult thing though, so maybe we should bring in the rest of the neurostore team on this?